changelog.txt

```diff
@@ -1,3 +1,13 @@
+4.19
+============
+New Features:
+**MSAL now adds telemetry for the region detected** See Issue [#2018](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2018)
```

Comment:
@neha-bhargava This issue doesn't have much detail. can you add a little more here?

Reply:
Discussed offline with @trwalke. Will add the details to document.